cmd/gaia/testnets/README.md

```diff
@@ -1,228 +1,185 @@
-You will require **Go 1.10+** for this tutorial.
+Install `go` by following the [official docs](https://golang.org/doc/install).
+**Go 1.10+** is required for the Cosmos SDK.
 
-#### Set GOPATH
+### Install Cosmos SDK
 
-First, you will need to set up your `GOPATH`. Make sure that the location `$HOME` is something like `/Users/<username>`, you can corroborate it by typing `echo $HOME` in your terminal.
-
-Go to `$HOME` with the command `cd $HOME` and open the the hidden file `.bashrc` with a code editor and paste the following lines \(or `.bash_profile` if your're using OS X\).
+Next, let's install the testnet's version of the Cosmos SDK. 
 
 ```
-export GOPATH=$HOME/go
-export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+go get github.com/cosmos/cosmos-sdk
+cd $GOPATH/src/github.com/cosmos/cosmos-sdk && git checkout v0.18.0
+make get_tools && make get_vendor_deps && make install
 ```
 
-Save and restart the terminal.
-
-_Note_: If you can't see the hidden file, use the shortcut `Command + Shift + .` in Finder.
+That will install the `gaiad` and `gaiacli` binaries. Verify that everything is OK:
 
-
-### Install [GNU Wget](https://www.gnu.org/software/wget/)
-
-**MacOS**
-
-```
-brew install wget
 ```
-
-**Linux**
-
-```
-sudo apt-get install wget
+gaiad version
+0.18.0-eceb56b7
 ```
 
-Note: You can check other available options for downloading `wget` [here](https://www.gnu.org/software/wget/faq.html#download).
-
-### Install Gaia
+### Node Setup
 
-Now we can fetch the correct versions of each dependency by running:
+Create the required configuration files:
 
 ```
-mkdir -p $GOPATH/src/github.com/cosmos/cosmos-sdk
-git clone https://github.com/cosmos/cosmos-sdk.git
-git checkout v0.18.0
-make get_tools // run $ make update_tools if already installed
-make get_vendor_deps
-make install
+gaiad init
 ```
 
-This will install the `gaiad` and `gaiacli` binaries. Verify that everything is OK by running:
+Name your node by editing the `moniker` in `$HOME/.gaiad/config/config.toml`. Note that only ASCII characters are supported. Using Unicode renders your node unconnectable.
 
 ```
-gaiad version
+# A custom human readable name for this node
+moniker = "<your_custom_name>"
 ```
 
-You should see:
+Your full node has been initialized! Please skip to [Genesis & Seeds](#genesis--seeds).
 
-```
-0.18.0-eceb56b7
-```
+## Upgrading From Previous Testnet
 
-And also:
+These instructions are for full nodes that have ran on previous testnets and would like to upgrade to the latest testnet.
 
-```
-gaiacli version
-```
+### Reset Data
 
-You should see:
+First, remove the outdated files and reset the data.
 
 ```
-0.18.0-eceb56b7
+rm $HOME/.gaiad/config/addrbook.json $HOME/.gaiad/config/genesis.json
+gaiad unsafe_reset_all
 ```
 
-## Full Node Setup
+Your node is now in a pristine state while keeping the original `priv_validator.json`. If you had any sentry nodes or full nodes setup before, they should continue to work.
```

Comment:
this preserves the config. the seeds/persistent-peers in the config will also need to upgrade if we're going to be able to connect with them